sessions/submission-acceptance-form.md

```diff
@@ -0,0 +1,44 @@
+# Confirm Participation 
+
+In order to confirm your participation in Our Networks 2019, please complete the below form by Monday July 29, 2019 (note: we use formspree.io to send responses).
+
+# Participation *
+
+- [ ] I/we will be presenting
+- [ ] I/we can no longer present
+
+# Session
+
+[dropdown?]
+
+- [ ] I have reviewed the description in GitHub
+
+# Presenter Details 
+
+Please confirm attendance and pronunciation
+
+
+Pronunciation: 
+Preferred pronouns:
+
+
+# Recording
+
+- [ ] The presentation can be livestreamed
+- [ ] The presentation can be recorded and posted on the Our Networks website
+
+You can see an example of the previous recorded talks at: https://ournetworks.ca/recorded-talks/
+If you have any questions or concerns about recording and livestreaming contact us: orga@ournetworks.ca
+
```

Comment:
Add line about option to withdraw consent after talk is recorded and before publishing.

Reply:
added